Rise-Vision/rise-vision-apps
# Open counter settings [stage-8]

## Description
Opens the ( empty ) Counter settings editor and adds the streamline _hourglass_ icon

## Motivation and Context
Counter component UI Settings development

## How Has This Been Tested?
Tested locally and on staging

https://apps-stage-8.risevision.com/templates/edit/851ba81e-1113-4788-994c-9d11a1f0c9a6/?cid=b428b4e8-c8b9-41d5-8a10-b4193c789443

**Note** Labels for the instances will be applied in the next PR as this involves applying appropriate label based on `type` of component

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
  - Manually tested
  - Initial automated unit tests added
  - Deployment will occur on Monday. Will monitor by testing production after released. There are no templates visible to users that use _rise-data-counter_ component in them, very low risk deployment
  - Rollback will involve re-running last master CCI build followed by reverting code changes in repo
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
- No documentation required
- Support not required
